File: CS/WindowsFormsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraRichEdit;
using DevExpress.XtraRichEdit.Services;

namespace WindowsFormsApplication1 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {            
            richEditControl1.ReplaceService<ISyntaxHighlightService>(new HTMLSyntaxHighlightService(richEditControl1));
            using(IRichEditDocumentServer server = richEditControl1.CreateDocumentServer()) {
                server.Text = "some HTML text";
                richEditControl1.Text = server.HtmlText;
            }
            richEditControl1.Document.DefaultCharacterProperties.FontName = "Consolas";
            richEditControl1.Document.DefaultCharacterProperties.FontSize = 9;
        }
    }
}

File: VB/WindowsFormsApplication1/Form1.vb
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraRichEdit
Imports DevExpress.XtraRichEdit.Services

Namespace WindowsFormsApplication1

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            richEditControl1.ReplaceService(Of ISyntaxHighlightService)(New HTMLSyntaxHighlightService(richEditControl1))
            Using server As IRichEditDocumentServer = richEditControl1.CreateDocumentServer()
                server.Text = "some HTML text"
                richEditControl1.Text = server.HtmlText
            End Using

            richEditControl1.Document.DefaultCharacterProperties.FontName = "Consolas"
            richEditControl1.Document.DefaultCharacterProperties.FontSize = 9
        End Sub
    End Class
End Namespace
